payments-core/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -488,7 +488,7 @@ class Stripe internal constructor(
     fun confirmPaymentIntentSynchronous(
         confirmPaymentIntentParams: ConfirmPaymentIntentParams,
         idempotencyKey: String? = null
-    ): PaymentIntent? {
+    ): PaymentIntent {
```

Comment:
Does this warrant an API change? It shouldn't affect users, except maybe they would get additional warnings?

Reply:
They’ll get an additional warning about a redundant null check.

Reply:
I do think this is worth adding to the changelog though. It's not a breaking change, but would be nice to call it out.